frontend/src/components/worksheets/Worksheet/Worksheet.js

```diff
@@ -303,8 +303,22 @@ class Worksheet extends React.Component {
         // The uuid are recorded by handleCheckBundle
         // Refreshes the checkbox after commands
         // If the action failed, the check will persist
+        const cmdMsgMap: { string: string } = { rm: ['deleting', 'deleted'] };
         let force_delete = cmd === 'rm' && this.state.forceDelete ? '--force' : null;
         this.setState({ updating: true });
+        let toastMsg =
+            (cmd in cmdMsgMap
+                ? Object.keys(this.state.uuidBundlesCheckedCount).length +
+                  ' bundles ' +
+                  cmdMsgMap[cmd][0]
+                : 'Executing ' + cmd + ' command') + '...';
+        const toastId = toast.info(toastMsg, {
```

Comment:
Can you add a comment explaining what this first toast is for?

Reply:
Added!